lib/agent0/bin/fuzz_bot_invariant_checks.py

```diff
@@ -231,6 +247,84 @@ def run_invariant_checks(
         raise FuzzAssertionException(*exception_message, exception_data=exception_data)
 
 
+def _lp_share_price(
+    block_number: BlockNumber,
+    interface: HyperdriveReadInterface,
+    test_epsilon: float,
+    pool_state: PoolState,
+    exception_message: list[str],
+    exception_data: dict[str, Any],
```

Comment:
Are you sure these are passed in via reference? Can we return the exception message and exception data just to make it explicit that these two fields are getting adjusted? Better yet, don't pass these in and add the message/data in the caller

Reply:
yeah lists and dicts are always passed by reference but i like what you're saying about making it more explicit anyway.